### ISSUE TITLE
Multi-lingual websites

### ISSUE THREAD

**Issue by ebeshero** [BMR59/Portu-WhatsApp#2]
We noticed in your meeting notes that you're considering whether and how best to have your site be directed to English and Portuguese speakers. Much depends on your fluency with Portuguese, but you could try writing each page in two different languages, and make each version available with JavaScript show/hide functions. That's not too difficult to code, but it might represent a lot of work for you in practice, so that's something to consider. @ghbondar can advise more on that! For now I wanted to show you one project that expresses itself in three languages (including Portuguese) that may be of interest: http://gl-pt.obdurodon.org/ . To be fair, the designer of this project didn't do this for her semester project--it's an ongoing part of her PhD dissertation work!  @BMR59 @pab124 @zme1

**Comment by ghbondar** [comment]
Indeed. Remember that this project needs to be more than just a digital bi-lingual edition of a text. It needs to answer a quantifiable research question using the computational methods that you will be learning in this course. I am concerned that, in effect, you are setting yourself up to complete two full projects, one in each language. As such, you would have half as much time to deeply probe and analyze the text in either one of them. Now, you are free to continue to develop your project after our course concludes, and to add an extensive translation then. But for our the project in our course, you might only want to have some samples of your translation interface, and save a complete version for post-course development. @BMR59 @pab124 @zme1 @ebeshero

**Comment by BMR59** [comment]
I had asked Zach about the logistics Portuguese option and we discussed that, if we did tackle it, it would be the very last thing we did. It would just be translation which I could do relatively quickly but it would all depend on how hencoding process . We were just mulling with the idea but we do have our research questions and will begin to sift through the text next week to help decide on one!
